### DISCUSSION TITLE
Tasmota Device Manager shows "Exception" for some modules

### DISCUSSION THREAD

**Discussion by NsinghP** [arendst/Tasmota#20014] [Q&A]
Hello, I noticed that Tasmota Device manger Shows an "Exception" for some of my installed Tasmota devices. (Please see the attached picture). Everything seems to be working without a glitch. Is it TDM bug or a Tasmota problem? Should I be concerned about this? and how do I figure out what the exception is?

Regards,
Navi.

![Screenshot (357)](https://github.com/arendst/Tasmota/assets/69470381/87a8eba9-ed41-4342-a845-709159e76034)

**Comment by sfromis** [comment]
Not TDM, and likely not Tasmota. Typically "Exception" is hardware-related crashes, but can sometimes be software too, depending on what you're doing. With 4 being roughly at the same time, power issues is a plausible consideration.

**Reply by NsinghP** [reply]
I'm not sure why the "Exception" happened. You may be right and this could just be power related. I have restarted all the devices that showed "Exception" tag and they are now clear. Since the devices are working normal and doing what they are supposed to do, I am not too concerned, but I will keep an eye on the devices and see if i can figure out what is triggering the "Exception". Thanks for your time. 
![Screenshot (358)](https://github.com/arendst/Tasmota/assets/69470381/40fc26c4-5354-42e1-8c5d-d0e0ae093bd4)
